src/common/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import CloseIcon from "@/images/ic_close.svg";
+import styles from "./badge.module.scss";
+import classNames from "classnames";
+
+interface BadgeProps extends React.HTMLAttributes<HTMLDivElement> {
+  color?: "blue" | "green" | "red";
+  mobile?: boolean;
+  close?: boolean;
+  onClick?: React.MouseEventHandler<HTMLDivElement>;
```

Comment:
아마 이렇게 표현해줘도 문제가 없을 것 같습니다!
```suggestion
  onClick?: () => void;
```